apis/facebook/api/src/objects/fb_post.e

```diff
@@ -0,0 +1,103 @@
+note
+	description: "Object representing a Post"
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name= Posts", "src=https://developers.facebook.com/docs/graph-api/reference/v2.9/post", "protocol=uri"
+class
+	FB_POST
+
+feature -- Access
+
+	id: detachable STRING
+		-- The post ID
+
+	caption: detachable STRING
+		-- Link caption in post that appears below name.
+		-- The caption must be an actual URLs and should accurately reflect the URL and associated advertiser or business someone visits when they click on it.
+
+	created_time: detachable STRING
+		-- The time the post was initially published. For a post about a life event, this will be the date and time of the life event.
+
+	message: detachable STRING
+		-- The status message in the post.
+
+	story: detachable STRING
+		-- Text from stories not intentionally generated by users, such as those generated when two people become friends, or when someone else posts on the person's wall.
+
+feature -- Change Element
+
+	set_id (a_id: like id)
+			-- Set `id' with `a_id'.
+		do
+			id := a_id
+		ensure
+			id_set: id = a_id
+		end
+
+	set_caption (a_caption: like caption)
+			-- Set `caption' with a `a_caption'.
+		do
+			caption := a_caption
+		ensure
+			caption_set: caption = a_caption
+		end
+
+	set_created_time (a_time: like created_time)
+			-- Set `created_time' with a_time.
+		do
+			created_time := a_time
+		ensure
+			created_time_set: created_time  = a_time
+		end
+
+	set_message (a_message: like message)
+			-- Set `message' with `a_message'.
+		do
+			message := a_message
+		ensure
+			message_set: message = a_message
+		end
+
+	set_story (a_story: like story)
+			-- Set `story' with `a_story'.
+		do
+			story := a_story
+		ensure
+			story_set: story = a_story
+		end
+
+feature -- Status Report
+
+	basic_out: STRING
```

Comment:
yes Facebook docs describe some fields as basic, so I decided to use `basic_out`, I will refactor it and use `debug_output` from `DEBUG_OUTPUT`